src/models/stores/serial.ts

```diff
@@ -0,0 +1,117 @@
+import { NodeChannelInfo } from "src/plugins/dataflow-tool/model/utilities/node";
+
+export class SerialDevice {
+    value: string;
```

Comment:
Our convention is two-space indents but this file has a mix of two- and four-space indents.

Reply:
thanks, fixed [here](https://github.com/concord-consortium/collaborative-learning/commit/9b1aa6833d63b670554005d766fbcc6a0f34518a)